share/templates/convert.json

```diff
@@ -10,9 +10,10 @@
 { "in": "02-Choral/05-SATB_Closed_Score_+_Organ.mscx",         "out": "02-Choral/05-SATB_Closed_Score_+_Organ.mscz"},
 { "in": "02-Choral/06-SATB_Closed_Score_+_Piano.mscx",         "out": "02-Choral/06-SATB_Closed_Score_+_Piano.mscz"},
 { "in": "02-Choral/07-Voice_+_Piano.mscx",                     "out": "02-Choral/07-Voice_+_Piano.mscz"},
-{ "in": "02-Choral/08-Barbershop_Quartet.mscx",                "out": "02-Choral/08-Barbershop_Quartet.mscz"},
-{ "in": "02-Choral/09-Liturgical_Unmetrical.mscx",             "out": "02-Choral/09-Liturgical_Unmetrical.mscz"},
-{ "in": "02-Choral/10-Liturgical_Unmetrical_+_Organ.mscx",     "out": "02-Choral/10-Liturgical_Unmetrical_+_Organ.mscz"},
+{ "in": "02-Choral/08-Barbershop_Quartet-(Men).mscx",          "out": "02-Choral/08-Barbershop_Quartet-(Men).mscz"},
+{ "in": "02-Choral/09-Barbershop_Quartet-(Women).mscx",        "out": "02-Choral/09-Barbershop_Quartet-(Women).mscz"},
```

Comment:
I guess the hyphen before `(Men)` or `(Women)` should be underscore instead?

Reply:
darn... thanks for catchint this

Reply:
fixed